ckanext/datarequests/public/datarequests.css

```diff
@@ -126,4 +163,7 @@
 .comment-new {
   border-top: 2px solid #f3f3f3;
   background-color: #fff;
-}
\ No newline at end of file
+}
+
```

Comment:
"All text files should end with a single blank line. This makes git commits easier to read since it's clearer what is being changed when lines are added to the end of a file and it avoids the verbose \ No newline at end of file warning in patches."

[https://www.drupal.org/docs/develop/standards/css/css-formatting-guidelines](url)

Reply:
Thank you. Fixing it